### PR TITLE
BLD: fix issue #218

### PR DESCRIPTION
**Pull request by lenak25**

### CODE REVIEW

File: test/jsonrpc_advanced_test.js [hunk 1]
```diff
@@ -133,6 +125,114 @@ describe('jsonrpc_advanced',()=>{
...
+      let signKey = await peerController.getNode().selfSubscribeAction();
+      await testUtils.sleep(1000);
+      const computeInput = await getComputeRequest(signKey);
+      client.request('sendTaskInput',computeInput,async (err,res)=>{
+        assert.strictEqual(true, res.result.sendTaskResult, "sendTaskResult not true");
+        await testUtils.sleep(1000);
+        client.request('getTaskResult',
+          {"taskId":computeInput.taskId},
+          async (err,res)=>{
+            if(err) assert.strictEqual(true,false,"err" + err);
```
Comment (AvishaiW) [inline test/jsonrpc_advanced_test.js]: Is this the way to send an error? doesn't look good

File: src/worker/controller/actions/tasks/HandleVerifiedTaskAction.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    const task = params.task;
+
+    const executeTaskCallback = () => {
+      this._controller.execCmd(constants.NODE_NOTIFICATIONS.EXEC_TASK, {task: task});
+    };
+
+    // First check if we are in the middle of PTT
+    // if so, schedule the task execution to after it is done
+    if (this._controller.principal().isInPTT()) {
+      this._controller.principal().once(constants.PTT_END_EVENT, executeTaskCallback);
```
[the remark this answers is not in the document]
Comment (lenak25) [reply src/worker/controller/actions/tasks/HandleVerifiedTaskAction.js]: Each task has its own callback

File: test/jsonrpc_advanced_test.js (hunk 1 above)
Comment (lenak25) [reply test/jsonrpc_advanced_test.js]: This will crash the UT

File: src/common/constants.js [hunk 3]
```diff
@@ -50,6 +50,7 @@ module.exports.NODE_NOTIFICATIONS = {
   'VERIFY_NEW_TASK': 'verifyreq', // request to perform verification of task
   'TASK_VERIFIED': 'tverified', // request to perform a deploySecretContract or computeTask tasks.
   'TASK_FINISHED': 'tfinished', // notify the task is finished, update network with result
+  'EXEC_TASK' : 'etask',
```
[the remark this answers is not in the document]
Comment (lenak25) [reply src/common/constants.js]: Done

**Pull request merged**